sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -114,53 +114,96 @@ namespace Azure { namespace Security { namespace Attestation {
    */
 
   class AttestationClient final {
+    /** @brief Construct a new Attestation Client object
```

Comment:
please don't hate me for this, but, can you explicitly add `private` here?

I know that's the default for `class`, but there's like a quick readability improvement when the eyes look at it w/o checking if this is a class or a struct.

I can live with it if you don't want to add it, but if it is not such a big deal for you, I would appreciate it :)

For example, I was not initially realizing that the constructors were becoming private because I usually look for either private or public at the first place (I know, that's my bad)